instrumentation/jmx-metrics/src/main/resources/jmx/rules/jvm.yaml

```diff
@@ -54,15 +54,45 @@ rules:
       ThreadCount:
         metric: count
         desc: Number of threads
+      DaemonThreadCount:
+        metric: daemon
+        desc: Number of daemon threads
   - bean: java.lang:type=OperatingSystem,*
-    prefix: jvm.cpu.
     type: gauge
     mapping:
+      TotalSwapSpaceSize:
+        metric: jvm.system.swap.space.total
+        desc: The host swap memory size in bytes
+        unit: by
+      FreeSwapSpaceSize:
+        metric: jvm.system.swap.space.free
+        desc: The amount of available swap memory in bytes
+        unit: by
+      TotalPhysicalMemorySize:
+        metric: jvm.system.physical.memory.total
+        desc: The total physical memory size in host
+        unit: by
+      FreePhysicalMemorySize:
+        metric: jvm.system.physical.memory.free
+        desc: The amount of free physical memory in host
+        unit: by
+      AvailableProcessors:
+        metric: jvm.system.available.processors
+        desc: The number of available processors
+        unit: "1"
+      SystemCpuLoad:
```

Comment:
From OTEL spec:
This metric is obtained from [com.sun.management.OperatingSystemMXBean#getSystemCpuLoad()](https://docs.oracle.com/en/java/javase/17/docs/api/jdk.management/com/sun/management/OperatingSystemMXBean.html#getSystemCpuLoad%28%29) on Java version 8..13, [com.sun.management.OperatingSystemMXBean#getCpuLoad()](https://docs.oracle.com/en/java/javase/17/docs/api/jdk.management/com/sun/management/OperatingSystemMXBean.html#getCpuLoad%28%29) on Java version 14+, and [com.ibm.lang.management.OperatingSystemMXBean#getSystemCpuLoad()](https://www.ibm.com/docs/api/v1/content/SSYKE2_8.0.0/openj9/api/jdk8/jre/management/extension/com/ibm/lang/management/OperatingSystemMXBean.html) on OpenJ9.

Should we also include `CpuLoad` to support higher Java versions?

Reply:
Added.

Reply:
Sorry to keep asking changes for this metric back and forth. 

I don't figure out a good way to solve the version problem in this case. It seems SystemCpuLoad is just marked as deprecated in version 14+. Will metrics be double counted if we add CpuLoad too?

What if SystemCpuLoad is totally removed in a future version? How will we use the same template to serve for customers who use versions that have both methods, and versions that have only CpuLoad method?

Reply:
That's fair. I see that `SystemCpuLoad` albeit deprecated is still able to be used on later versions so I decided to remove `CpuLoad` for now to avoid double metrics.

However, we will run into an issue if `SystemCpuLoad` is totally removed. So, we can consider using the `runtime_telemetry` library as you suggested in the future.

Though, for this initial release, this should be sufficient.

Reply:
Can you add a comment or add this summary to README file?